tools/nemo_forced_aligner/utils/make_ctm_files.py

```diff
@@ -105,7 +106,17 @@ def make_ctm(
                     # replace any spaces with <space> so we dont introduce extra space characters to our CTM files
                     text = text.replace(" ", SPACE_TOKEN)
 
-                    f_ctm.write(f"{utt_obj.utt_id} 1 {start_time:.2f} {end_time - start_time:.2f} {text}\n")
+                    ctm_line = get_ctm_line(
+                        source=utt_obj.utt_id,
+                        channel=1,
+                        beg_time=start_time,
+                        duration=end_time - start_time,
+                        token=text,
+                        conf=None,
+                        type_of_token='lex',
+                        speaker=None,
```

Comment:
@tango4j  Could you please add `output_precision=2`? Since we use a spectrogram window shift of 0.01s, there's no way currently to get an alignment to a precision higher than 0.01 seconds.

Reply:
Updated to default value of 2. Also changed the test functions to pass it.